frontend/packages/ceph-storage-plugin/integration-tests/tests/ocp-tests/installFlow.scenario.ts

```diff
@@ -0,0 +1,84 @@
+import { execSync } from 'child_process';
+import { browser, $, ExpectedConditions as until } from 'protractor';
+import * as crudView from '@console/internal-integration-tests/views/crud.view';
+import { MINUTE, SECOND, NS } from '../../utils/consts';
+import {
+  selectWorkerRows,
+  ocsOperatorStatus,
+  namespaceDropdown,
+} from '../../views/installFlow.view';
+import { goToInstalledOperators, getStorageClusterLink } from '../../views/add-capacity.view';
+import { click } from '@console/shared/src/test-utils/utils';
+import { waitFor } from '../../utils/helpers';
+
+const btnCreate = $('.pf-m-primary');
+const namespaceLink = $('#openshift-storage-link');
+const filterItem = $('input[data-test-id="item-filter"]');
+const OCS_OPERATOR = $('a[data-test-operator-row="OpenShift Container Storage"]');
+
+const LABEL_KEY = 'cluster.ocs.openshift.io/openshift-storage';
+const createLink = $('.pf-c-card__footer a');
+const sizeDropdown = $('button[id="ocs-service-capacity-dropdown"]');
+const optionSmallSize = $('button[id="512Gi-link"]');
+
+describe('Testing OCS Subscription', () => {
+  beforeAll(() => {
+    execSync(
+      'kubectl apply -f https://raw.githubusercontent.com/openshift/ocs-operator/master/deploy/deploy-with-olm.yaml',
+    );
+  }, 2 * MINUTE);
+
+  // Once other tests are pushed remove this
+  afterAll(() => {
+    execSync(`kubectl delete storageclusters ocs-storagecluster -n openshift-storage`);
+    execSync(
+      'kubectl delete -f https://raw.githubusercontent.com/openshift/ocs-operator/master/deploy/deploy-with-olm.yaml',
+    );
+  }, 2 * MINUTE);
+
+  it(
+    'Test Installation for OCS operator suceeds or not',
+    async () => {
+      await goToInstalledOperators();
+      await click(namespaceDropdown);
+      await click(namespaceLink);
+      await filterItem.sendKeys('OCS');
+      await waitFor(ocsOperatorStatus, 'Succeeded', 5);
+      await click(OCS_OPERATOR);
+      await browser.wait(until.and(crudView.untilNoLoadersPresent));
+      await browser.wait(until.visibilityOf(createLink), 10 * SECOND);
+      const storageClusterLink = await getStorageClusterLink();
+      await click(storageClusterLink);
+      await browser.wait(until.and(crudView.untilNoLoadersPresent));
+      const nodes = await selectWorkerRows();
```

Comment:
It should be `selectSchedulableNodesRows` as even worker node can be non-schedule-able.

Reply:
then it shouldn't be listed there if that is the case. Which I can verify later as an expectation when I check for labels on that particular node.

Reply:
Have added the checks for worker and taints. If such nodes are indeed gettings selected then the test will fail.